src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1563,26 +1563,16 @@ public static bool HasNames(this TupleExpressionSyntax tuple)
             {
                 // for ( |
                 // foreach ( |
+                // await foreach ( |
                 // using ( |
+                // await using ( |
```

Comment:
these comments seem wrong.

Reply:
I didn't follow. The comments seem correct.